raiden-dapp/src/components/SplashScreen.vue

```diff
@@ -188,12 +195,6 @@ export default class Loading extends Vue {
     height: 48px;
   }
 
-  &__disclaimer {
-    margin: 0 auto;
-    margin-top: 180px;
-    max-width: 560px;
```

Comment:
Thank you for taking care of this. I'm just wondering why the max-width is removed? I added it to have the text break in those two lines if the window is large.

Reply:
The text already broke, but the max width made the text appear uncentered

Reply:
Ah, ok, did not notice that. Thank you.